servers/display_server.cpp

```diff
@@ -1602,6 +1643,12 @@ void DisplayServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_native_icon", "filename"), &DisplayServer::set_native_icon);
 	ClassDB::bind_method(D_METHOD("set_icon", "image"), &DisplayServer::set_icon);
 
+	ClassDB::bind_method(D_METHOD("mouse_button", "x", "y", "mouse_button_index", "pressed", "double_click", "cancelled", "window"), &DisplayServer::mouse_button);
+	ClassDB::bind_method(D_METHOD("mouse_motion", "prev_x", "prev_y", "x", "y", "window"), &DisplayServer::mouse_motion);
+	ClassDB::bind_method(D_METHOD("touch_press", "idx", "x", "y", "pressed", "double_click", "window"), &DisplayServer::touch_press);
+	ClassDB::bind_method(D_METHOD("touch_drag", "idx", "prev_x", "prev_y", "x", "y", "pressure", "tilt", "window"), &DisplayServer::touch_drag);
+	ClassDB::bind_method(D_METHOD("key", "key", "char", "unshifted", "physical", "modifiers", "pressed", "window"), &DisplayServer::key);
+
```

Comment:
I'm very skeptical about handling input through a DisplayServer. Can you provide some context why this is needed and can't use the usual input handling route?